apriltag_ros/msg/AprilTagDetection.msg

```diff
@@ -11,4 +11,11 @@ float64[] size
 # tag, the homography is from the four tag corners. If a tag bundle, the
 # homography is from at least the four corners of one member tag and at most the
 # four corners of all member tags.
-geometry_msgs/PoseWithCovarianceStamped pose
\ No newline at end of file
+geometry_msgs/PoseWithCovarianceStamped pose
+
+# Tag center in pixel coordinates
+apriltag_ros/Point2D tag_center
+
+# Detected tag corners in pixel coordinates. Starting from bottom left going 
+# around anti-clockwise
```

Comment:
This bottom left is the _tag's_ bottom left, not the bottom left in the camera frame, right?

Reply:
Yes. Consider 
![image](https://user-images.githubusercontent.com/10868851/133124867-70ba9ef7-2295-4fe8-9a05-0bba17bcb06d.png) where the tags are upside down 
![image](https://user-images.githubusercontent.com/10868851/133124930-bcece6d7-50ba-4d16-afcc-c99fe07f66f2.png)